packages/framework/src/Controller/Admin/DefaultController.php

```diff
@@ -326,4 +326,22 @@ public function scheduleCronAction(string $serviceId): Response
 
         return $this->redirectToRoute('admin_default_dashboard');
     }
+
+    /**
+     * @Route("/cron/switch/{serviceId}/{action}", requirements={
+     *     "action": "enable|disable"
+     * }))
+     * @param string $serviceId
+     * @param string $action
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function switchCronAction(string $serviceId, string $action): Response
```

Comment:
Is the `$action` argument necesasry? I would not use it at all - what do you think?